packages/frontend/src/Examples.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+import type { Example, Category } from '../../shared/src/models/examples';
+import { onMount } from 'svelte';
+import { NavPage } from '@podman-desktop/ui-svelte';
+import { bootcClient } from './api/client';
+import ExamplesCard from './lib/ExamplesCard.svelte';
+
+let groups: Map<Category, Example[]> = new Map();
+
+const UNCLASSIFIED: Category = {
+  id: 'unclassified',
+  name: 'Unclassified',
+};
+
+onMount(async () => {
+  // onmount get the examples
+  let examples = await bootcClient.getExamples();
```

Comment:
it means that it's only when we create the component, it won't be reactive it's ok ?

Reply:
okay to not be reactive as we as not making any changes to the assets file reactively / modifying it / or getting it from a local file.